### ISSUE TITLE
Hacktoberfest 2020 summary

### ISSUE THREAD

**Issue by eshellman** [EbookFoundation/free-programming-books#5033]
This repo received 1,314 @Hacktoberfest PRs and merged 559 (39%) of them (merges from 304 individuals)! Thanks everybody! @saritip, @GintasS, @LeoTino, @kadhirash, @ch-a-nd-a-n, @chessboards, and @harsh-kk had 6 or more commits merged. Thanks everyone, and I hope many of you didn't just do it for the t-shirts.

We created a cheatsheet page, added several course pages, several translated "ABOUT" and "CONTRIBUTING" pages, and vastly expanded our Indonesian pages.

Now that Hacktoberfest is over, it's time to tell the world! Post to social media, tell your friends, and brag about how you are part of one of the top ten starred repos out of tens of millions on GitHub (you did ⭐  the repo, didn't you?)

But don't stop there. Your contributions to Open Source benefit the whole world. For many of the 304 hacktoberists, this was a first contribution. Don't let it be their last!

If we didn't merge your commit, please forgive the very small team that triaged, reviewed, and merged the contributions. We had no idea we would get so many contributions. We can improve our documentation and our automation based on what we learned. Come back next year. Come back this month. Come back tomorrow. 

Also consider sponsoring us. We are a charitable non-profit based in New Jersey, USA, Planet Earth.

**Comment by CH-ND-N** [comment]
Surely sir I am on my way to my nest repo.

**Comment by GintasS** [comment]
I personally did it for the tree, way better than a t-shirt in my opinion. Thanks for you and the team for reviewing PRs!

**Comment by kadhirash** [comment]
It was a wonderful experience! Looking to contribute more and help clean the PR's! 😄

**Comment by eshellman** [comment]
Update: Together with a small team of volunteers, we were able to review every one of the 1314 PRs from October. Some of the suggestions were not appropriate to the project, but now that the dust has settled, we have accepted 55% of the PRs submitted during Octoberfest. The 338 open-source enthusiasts whose changes we accepted come from all over the world, and for many of them, these represented their first ever contributions to open source. We sincerely hope that they will continue to enrich the world's information commons.

Free-Programming-Books now contains 3,770 links to free books, 1,254 free courses, and 822 other resources. There are lists of books in 35 spoken languages and courses in 20 of them. The list covers over 200 programming languages and platforms. October saw major expansions of our lists for Indonesian, Hindi, Tamil, and Portuguese readers.
